plugins/android_boot/README.md

```diff
@@ -0,0 +1,51 @@
+# Android Bootloaders
+
+## Introduction
+
+This plugin is used to update hardware that use partitions to store their firmware on.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob as a Raw Disk File
+in the IMG format. The firmware blob will be flashed to the partition.
+
+This plugin supports the following protocol ID:
+
+* be.dylanvanassche.android_boot
```

Comment:
If this plugin is just basically "dd", then maybe  protocol should be `org.fwupd.android_boot` (or maybe even it shouldn't be called `android_boot` but `dd`?)

Reply:
Or maybe even `com.google.android_boot`?

Reply:
It does `dd` with some additional stuff like Android A/B bootslots.
I rather name it then `org.fwupd.android_boot` or `org.postmarketos.android_boot` --> https://postmarketos.org/

Reply:
I think com.google.android_boot is probably best IMHO.

Reply:
Renamed locally already to `com.google.android_boot` as this suggestion came up twice already.